go.mod

```diff
@@ -15,10 +15,13 @@ require (
 	github.com/pborman/uuid v1.2.1
 	github.com/robfig/cron v1.2.0
 	github.com/stretchr/testify v1.8.0
-	go.temporal.io/api v1.11.0
+	go.temporal.io/api v1.11.1-0.20220804232755-77e12eef4f2d
```

Comment:
Is this implemented in the server? If not, I think we should wait until then to merge this. Also, I wonder if once implemented in the server we should tag the API since this is such a big change.

Reply:
The server has dummy implementations returning `Unimplemented` error.

Reply:
Why not wait until implemented there to add here? Can reference a local SDK perfect path during development.

Reply:
I added those to unblock myself since I made changes to the API (same with this PR). Cc: @yux0

Reply:
The server implementation PR (for batch API) will be land soon, but we should not block this PR just because server currently return unimplemented as this API is not used anywhere anyway. Server also want to use UpsertMemo feature which is currently blocked on this PR.

Reply:
Just to be clear, this PR only updates the generated files (`client/service_proxy.go` and `converter/grpc_interceptor.go`). It does not create/expose an API that call those unimplemented API endpoints.

Reply:
> but we should not block this PR just because server currently return unimplemented

Why not? This is a user-facing API update to a non-tagged API version.

> Server also want to use UpsertMemo feature which is currently blocked on this PR.

This should not block development. I can show how to use `go.mod` replace clauses to reference a local form of this while you develop. If server wants to use `UpsertMemo`, that's blocked by #883 where I've also requested we have real tests to even confirm it works. All of this work can be done locally.

Reply:
I meant that this PR blocks merging the `UpsertMemo` PR. I've done all the  local tests, and also almost done with writing the integration tests there.

Reply:
:+1: Yeah, integration tests means things are working which works for me. I'd happily approve/merge. Curious, why does this PR block that one?

Reply:
The changes to API related to batch operations affect the generated files `grpc_interceptor.go` and `service_proxy.go`. This PR just re-generates those files. It has nothing to do with `UpsertMemo`, but since my changes also affect those files, I thought it would better to have a PR doing an update for batch related stuff first instead of altogether.

Reply:
Sure, you're call, updating API is updating API, could do in same PR or other.

So I can merge this. My concern is that we are giving new public API to end users that is not tagged and is not implemented. But as we've done with other API changes, we can just hope they don't try to use it yet.